### DISCUSSION TITLE
3.02

### DISCUSSION THREAD

**Discussion by horsicq** [horsicq/DIE-engine#33] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/horsicq/DIE-engine/releases/tag/3.02'>3.02</a>.</em>

**Comment by class101** [comment]
Arch Linux based distros users

you can install it with

```bash
paru -S detect-it-easy-git
```
This will compile all from sources and install in the system.

**Comment by B0u3Zizi** [comment]
where is version lite ? please add it

**Reply by horsicq** [reply]
Thanks for you responce! I will think about it.
